app-simple-wallet/app/src/main/java/com/goldenraven/devkitwallet/data/Wallet.kt

```diff
@@ -31,39 +31,33 @@ object Wallet {
     }
 
     private fun initialize(
-        externalDescriptor: String,
-        internalDescriptor: String,
+        descriptor: Descriptor,
+        changeDescriptor: Descriptor,
     ) {
         val database = DatabaseConfig.Sqlite(SqliteDbConfiguration("$path/bdk-sqlite"))
         wallet = BdkWallet(
-            externalDescriptor,
-            internalDescriptor,
-            // Network.REGTEST,
+            descriptor,
+            changeDescriptor,
             Network.TESTNET,
-            database,
+            database
         )
     }
 
     fun createBlockchain() {
-        blockchainConfig = BlockchainConfig.Electrum(ElectrumConfig(electrumURL, null, 10u, 20u, 10u))
-        // blockchainConfig = BlockchainConfig.Esplora(EsploraConfig(esploraUrl, null, 5u, 20u, 10u))
+        val blockchainConfig = BlockchainConfig.Electrum(ElectrumConfig(electrumURL, null, 5u, null, 10u, true))
```

Comment:
Let's use the named parameter approach here so as to make the arguments explicit. Something like
```kotlin
ElectrumConfig(
    url = electrumURL, 
    socks5 = null, 
    retry = 5u, 
    timeout = null, 
    stopGap = 10u,
    validateDomain = true
)
```